src/activities/act-7-2-1.xml

```diff
@@ -91,9 +91,45 @@
       <ol label="a.">
         <li>
           <p>
-            Answers for each of the prompts above.
-          </p>
+            When <m>y \lt  4</m>, <m>y</m> is an increasing function of <m>t</m>.  When <m>y \gt 4</m>, <m>y</m> is a decreasing function of <m>t</m>.
+          </p>              
+          <figure xml:id="F-act-7-2-1-a">
```

Comment:
Here's another instance where `figure` should be replaced by `sidebyside` (having neither `caption` nor `@xml:id`).